auth0-lock/index.d.ts

```diff
@@ -103,22 +103,36 @@ interface Auth0LockConstructorOptions {
     usernameStyle?: string;
 }
 
+interface Auth0LockFlashMessageOptions {
+    type: "success" | "error";
+    text: string;
+}
+
+interface Auth0LockShowOptions {
+    allowedConnections?: string[];
+    allowSignUp?: boolean;
```

Comment:
Do you need   allowLogin as well as per https://github.com/auth0/lock#showoptions